MonoGame.Framework/Audio/SoundEffect.cs

```diff
@@ -150,6 +152,8 @@ public static int GetSampleSizeInBytes(TimeSpan duration, int sampleRate, AudioC
         {
             if (sampleRate < 8000 || sampleRate > 48000)
                 throw new ArgumentOutOfRangeException();
+            if (duration < TimeSpan.Zero)
+                throw new ArgumentOutOfRangeException();
 
             // Reference: http://social.msdn.microsoft.com/Forums/windows/en-US/5a92be69-3b4e-4d92-b1d2-141ef0a50c91/how-to-calculate-duration-of-wave-file-from-its-size?forum=winforms

```

Comment:
FYI.  Likely to be some conflicts with https://github.com/mono/MonoGame/pull/4469 in this file depending on which is merged first.